.azure-pipelines/linux-conda-CI.yml

```diff
@@ -239,8 +239,8 @@ jobs:
     displayName: 'Install environment'
 
   - script: |
-      conda install "numpy$(numpy.version)" "scipy$(scipy.version)"
-    displayName: 'install numpy, scipy'
+      conda install "numpy$(numpy.version)" "scipy$(scipy.version)" "sympy$(sympy.version)"
```

Comment:
Why do you need sympy?

Reply:
When run CI Pipeline, it tells need sympy, otherswise it will be failed.